components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
+  version: "0.0.2",
+  props: {
+    reddit,
+    timer: {
+      label: "Polling schedule",
+      description:
+        "Pipedream poll for new links and emit accordingly with this schedule.",
```

Comment:
I'd change to: "Pipedream polls for new links on this schedule"

Reply:
Small question regarding the dot, or punctuation mark.
In this documentation for object descriptions, we don't use the dot?

`Pipedream polls Reddit for new links on this schedule`
better than
`Pipedream polls Reddit for new links on this schedule**.**`
?

So in actions I had this question and I defaulted to not using the dot in actions descriptions but using in the parameter description.

Reply:
In descriptions that are full sentences, like this, I normally put a period at the end. If the description is a phrase, e.g. "The subreddit to watch for new links", I don't include a period.

Reply:
Nice criteria. I will follow along and use it.